proxy-robots-module/src/main/java/orwell/proxy/robot/IPWebcam.java

```diff
@@ -13,12 +13,12 @@
     private final URL url;
 
     public IPWebcam(final URL url) {
-        assert(null != url);
+        assert (null != url);
```

Comment:
assert is a language feature, not a function. But in a way, you are right indeed, parenthesis are confusing and useless (a bit like adding parenthesis after a return () ).
I will remove parenthesis in such assertions.